contiamo/data.py

```diff
@@ -137,10 +139,31 @@ def discover(self, dataframe=None, filename=None, include_index=False):
         url = self.url_template.format(action='upload/discover')
         return self._post_data(url, dataframe, filename, include_index)
 
-    def upload(self, dataframe=None, filename=None, include_index=False):
+    def upload(self, dataframe=None, filename=None, include_index=False, chunk_size=100000):
```

Comment:
as discussed, let's default to `len(dataframe)`

Reply:
Is `None` or `0` a better default argument? (I'm hijacking this PR to discuss fine Python nuances... ;)

Reply:
I think `None`. That often implies that some default will be used.